socket_connection:
  port_num: 34578
  connect_to: robot-helper
